api/mapping/views.py

```diff
@@ -210,56 +240,62 @@ def form_valid(self, form):
         return super().form_valid(form)
 
     def get_success_url(self):
-        scan_report_field = ScanReportField.objects.get(
-            pk=self.kwargs.get('pk')
-        )
+        scan_report_field = ScanReportField.objects.get(pk=self.kwargs.get("pk"))
 
-        return "{}?search={}".format(reverse('fields'), scan_report_field.scan_report_table.id)
+        return "{}?search={}".format(
+            reverse("fields"), scan_report_field.scan_report_table.id
+        )
```

Comment:
another example of random changing of single line to multiple lines

Reply:
Yes, think this was Python Black again....